.github/workflows/reusable-performance.yml

```diff
@@ -29,7 +29,8 @@ on:
     secrets:
       CODEVITALS_PROJECT_TOKEN:
         description: 'The authorization token for https://www.codevitals.run/project/wordpress.'
-        required: true
+        required: false
+        default: ''
```

Comment:
The addition of a `default` here caused the workflow to fail as invalid ([ref](https://github.com/WordPress/wordpress-develop/actions/runs/11295473849/workflow)). It seems that [the workflow syntax ](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onworkflow_callsecrets)does not support default values for secrets.

```suggestion
```